packages/@react-aria/autocomplete/src/useAutocomplete.ts

```diff
@@ -35,14 +35,12 @@ export interface AriaAutocompleteProps extends AutocompleteProps {
 
 export interface AriaAutocompleteOptions extends Omit<AriaAutocompleteProps, 'children'> {
   /** The ref for the wrapped collection element. */
-  collectionRef: RefObject<HTMLElement | null>,
-  /** The ref for the wrapped input element. */
-  inputRef: RefObject<HTMLInputElement | null>
+  collectionRef: RefObject<HTMLElement | null>
```

Comment:
wondering if this could move to a third parameter now that there is only one, but fine for this release

Reply:
sounds good, I'll make a note and handle this in a follow up